ui.md

```diff
@@ -16,7 +16,7 @@ Sometimes, when designing a tool or a bookmarklet for non-dev people, you might
 
 *Example*
 
-You want to design a *à la carte* Google Scholar scraper for researchers.
+You want to design an *à la carte* Google Scholar scraper for researchers.
```

Comment:
I wasn't sure either.  But this exchange seems to say to use "an"
http://english.stackexchange.com/questions/246158/should-i-use-a-or-an-before-a-foreign-word-that-starts-with-a-u

I think the French pronounce the à with a vowel sound.  Since this a a loaned term we should use the 'an' article.